submissions/NadiaVorontsova/hooli-style-popup/index.html

```diff
@@ -10,223 +10,159 @@
                                 </li>
-                                <li class="header__popup__menu__item">
-                                    <a href="#" class="header__popup__menu__link link">
-                                        <div class="header__popup__menu__container">
-                                            <img src="assets/translation.png" width="48" height="35" alt=""
-                                                class="header__popup__menu__img">
-                                            <p>Translate</p>
-                                        </div>
+                                <li class="header__popup_item">
+                                    <a href="#" class="header__popup_link link">
+                                        <img src="assets/translation.png" width="48" height="35" alt=""
+                                            class="header__popup__img">
+                                        <p>Translate</p>
                                     </a>
                                 </li>
-                                <li class="header__popup__menu__item">
-                                    <a href="#" class="header__popup__menu__link link">
-                                        <div class="header__popup__menu__container">
-                                            <img src="assets/videos.png" width="48" height="35" alt=""
-                                                class="header__popup__menu__img">
-                                            <p>Videos</p>
-                                        </div>
+                                <li class="header__popup_item">
+                                    <a href="#" class="header__popup_link link">
+                                        <img src="assets/videos.png" width="48" height="35" alt=""
+                                            class="header__popup__img">
+                                        <p>Videos</p>
                                     </a>
                                 </li>
-                                <li class="header__popup__menu__item">
-                                    <a href="#" class="header__popup__menu__link link">
-                                        <div class="header__popup__menu__container">
-                                            <img src="assets/search.png" width="48" height="35" alt=""
-                                                class="header__popup__menu__img">
-                                            <p>Search</p>
-                                        </div>
+                                <li class="header__popup_item">
+                                    <a href="#" class="header__popup_link link">
+                                        <img src="assets/search.png" width="48" height="35" alt=""
+                                            class="header__popup__img">
+                                        <p>Search</p>
```

Comment:
[Check your markup](https://caninclude.glitch.me/caninclude?child=p&parent=a). Tag `p` inside `a` not a good decision)

Reply:
i decided to remove tag p in tag a, is that okey?

Reply:
Yes, it's ok, also, you can replace `p` with `span` as you need. The main point is to check your markup, one of the most popular tools for this is [W3C validator](https://validator.w3.org/)